README.md

```diff
@@ -6,6 +6,9 @@ A library to **sub**mit e**xt**rinsics to a [substrate](https://github.com/parit
 
 See [examples](./examples).
 
+If you use `#[derive(Call)]` without `#[module]` in the same module, you will get errors
+complaining about an undefined method with a name starting with `with_`.
```

Comment:
Please open an issue for this, or provide some context here

Reply:
It took me a long time to figure this out, so I added this for the benefit of others.

Reply:
I've raised an issue for it to improve the error message: https://github.com/paritytech/substrate-subxt/issues/170. But probably we should remove from the readme since it doesn't make sense without any context.